Code/Light.GuardClauses.Tests/ComparableAssertions/RangeTests.cs

```diff
@@ -49,4 +54,76 @@ public static void ConstructorException(int from, int to)
         act.Should().Throw<ArgumentOutOfRangeException>().And
            .Message.Should().Contain($"{nameof(to)} must not be less than {from}, but it actually is {to}.");
     }
+
```

Comment:
An additional unit test that asserts that `Range.For(IEnumerable enumerable)` throws when `enumerable` is `null` would be good to have.